doc/source/reference/drawing.rst

```diff
@@ -4,6 +4,21 @@
 Drawing
 *******
 
+NetworkX provides basic functionality for visualizing graphs, but its main
+goal is to enable graph analysis rather than perform graph visualization. In
+the future, graph visualization functionality may be removed from NetworkX or
+only available as an add-on package.
+
+Proper graph visualization is hard, and we highly recommend that people
+visualize their graphs with tools dedicated to that task. Notable examples
+of dedicated and fully-featured graph visualization tools are
+`Cytoscape <http://www.cytoscape.org/>`_ and
+`Graphviz <http://www.graphviz.org/>`_, and for the those of you seeking
+automated layout algorithms with `LaTeX <http://www.latex-project.org/>`_
+styling, `PGF/TikZ <http://sourceforge.net/projects/pgf/>`_.
+To use these and other such tools, you should export your NetworkX graph
+into a format that can be read by those tools.
```

Comment:
Will this not depend on what features they want in, since not every export format is equivalent. However, I can recommend graphml for Cytoscape...so maybe I'll do just that.